### ISSUE TITLE
False positive in `function_left_parentheses_linter()`

### ISSUE THREAD

**Issue by IndrajeetPatil** [r-lib/lintr#1963]
``` r
'roxygen2::roc_proc_text(paste(roxygen, collapse = "\n"))[[1L]]$get_section("examples")' -> code

lintr::lint(text = code, linters = lintr::function_left_parentheses_linter())
#> <text>:1:1: style: [function_left_parentheses_linter] Left parenthesis should be on the same line as the function's symbol.
#> roxygen2::roc_proc_text(paste(roxygen, collapse = "
#> ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

<sup>Created on 2023-04-17 with [reprex v2.0.2](https://reprex.tidyverse.org)</sup>

**Comment by lorenzwalthert** [comment]
Also with lines broken (not sure that matters?):
```r
text <- roxygen2::roc_proc_text(
  roxygen2::rd_roclet(),
  paste(roxygen, collapse = "\n")
)[[1L]]$get_section("examples")
```
